packages/stacked_crashlytics/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [0.0.1] - TODO: Add release date.
```

Comment:
Change this to 0.1.0 and remove the text on the right

For the Entry you can add

- Initial implementation of Crashlytics Service